### PR TITLE
Fixes squeaking

### PR DESCRIPTION
**Pull request by Kyep**
:cl: Kyep
fix: Dead mice no longer squeak.
fix: Ghosts walking over living mice no longer makes them squeak.
/:cl:

Q: What happens when people arrange a row of dead mice on the bridge, then organize a bunch of ghosts to constantly run back and forth over them at warp speed, generating an unholy undead mouse squeak opera?
A: This PR.

### CODE REVIEW

**Comment by Citinited** [comment]
Would the clown jumpsuit also be affected by this?

**Comment by variableundefined** [comment]
I am looking into how to fix this properly because I am certain this doesn't happen on tg without any need for this kind of check. However I was busy yesterday and did not have time to perform a thorough test.

File: code/modules/mob/living/simple_animal/friendly/mouse.dm [hunk 1]
```diff
@@ -91,9 +90,10 @@
 
 /mob/living/simple_animal/mouse/Crossed(AM as mob|obj)
 	if(ishuman(AM))
-		if(!stat)
+		if(stat == CONSCIOUS)
 			var/mob/M = AM
 			to_chat(M, "<span class='notice'>[bicon(src)] Squeek!</span>")
+			M << 'sound/effects/mousesqueek.ogg'
```
Comment (variableundefined) [inline code/modules/mob/living/simple_animal/friendly/mouse.dm]: SEND_SOUND Macro please.

File: code/modules/mob/dead/dead.dm [hunk 2]
```diff
@@ -0,0 +1,25 @@
+/mob/dead/forceMove(atom/destination)
```
Comment (variableundefined) [inline code/modules/mob/dead/dead.dm]: See instructions on discord

File: code/modules/mob/living/simple_animal/friendly/mouse.dm (hunk 1 above)
Comment (Kyep) [reply code/modules/mob/living/simple_animal/friendly/mouse.dm]: Changed

File: code/modules/mob/dead/dead.dm (hunk 2 above)
Comment (Kyep) [reply code/modules/mob/dead/dead.dm]: Changed

**Comment by Kyep** [comment]
@Citinited No.

**Pull request merged**